src/core/Geocoding.js

```diff
@@ -114,16 +114,22 @@ module.exports = function Geocoding(options) {
         async: false,
         url: url
     });
-    onComplete = onComplete || function onComplete(geometry) {
-      $("#lat").val(geometry.lat);
-      $("#lng").val(geometry.lng);
-
-      map.setView([geometry.lat, geometry.lng], options.zoom);
+    onComplete = onComplete || function onComplete(response) {
+      if(response.status === "OK") {
+        let geometry = response.results[0].geometry.location;
```

Comment:
This is what's causing the tests to fail. I've looked for any other instance that is calling this function and why it might be throwing errors but I can't see anything. It's only the tests, for some reason this is causing the tests to not build the BlurredLocation object.